### ISSUE TITLE
Get the list, only 100 pieces of data will be displayed, out of the 200 pieces I filled out

### ISSUE THREAD

**Issue by huazhanhao950327** [facundoolano/app-store-scraper#206]
* Operating System:
* Node version:
* app-play-scraper version:

### Description: 
Describe your issue here

### Example code:
```js
// Put code to reproduce the issue here
```

### Error message:

```
Put error message here
```

**Comment by ilkerb** [comment]
Operating System: macOS 14.6.1
Node version: 20.18.1
app-play-scraper version: 0.18.0

I have similar issue, list function returns first 100 results no matter what limit value given above 100. I tried to paginate but that didn't helped either. It reacts to limit parameter for values below 100 without any problems but it does not return more than 100 results no matter which value given above 100. If this is related to a change on the backend this function use, please correct the documentation so 100 is the limit for list, not 200. If not (I wish it is not) please lead me on how to get all 200 results for a list call.

```
store.list({
  collection: store.collection.TOP_GROSSING_IOS,
  category: store.category.PHOTO_AND_VIDEO,
  num: 200
}).then(console.log).catch(console.log);
```

**Comment by learneryus** [comment]
do you finish the problem? I also have the question

**Comment by ilkerb** [comment]
> do you finish the problem? I also have the question

nope, I couldn't manage to get more than 100 with this, I finally reached more than 100 apps with web scraping

**Comment by alexflm** [comment]
@ilkerb  Hello. Just wondering, did you find a way to get a list of more than 100 applications from the first source or from aso/analytics services?

**Comment by ilkerb** [comment]
@alexflm no I couldn't make it through the first source even with this library or other libraries that I found. I am scraping them through a website that displays longer top app charts for App Store.

**Comment by learneryus** [comment]
@ilkerb can you share the website? If you can share your code which i want to use to finish my paper for graduation
 ,i will very grateful。

**Comment by Remember2015** [comment]
same problem, need help +1
